packages/plugin-dom-layout/src/DomReconciliationEngine.ts

```diff
@@ -112,7 +112,13 @@ export class DomReconciliationEngine {
                     const ids = this._renderedNodes.get(linkedNode);
                     if (ids) {
                         for (const id of ids) {
-                            if (!oldObjects.has(id)) {
+                            if (!this._objects[id]) {
+                                console.warn(
+                                    'A rendering object are remove twice, one of the references was wrong.',
```

Comment:
Le rendu cible a déjà été effacé. Il existe donc plus que une référence vers cet élément, et donc il y a au moins une reférence erronnée.

Reply:
Trying to remove a rendering object that has already been removed. One of the references is wrong.